src/applications/vaos/appointment-list/pages/UpcomingAppointmentsDetailsPage/index.unit.spec.js

```diff
@@ -422,7 +422,7 @@ describe('VAOS Page: ConfirmedAppointmentDetailsPage with VAOS service', () => {
         ).to.be.ok;
       });
     });
-    it('should show custom error message for bad appointment Id', async () => {
+    it.skip('should show custom error message for bad appointment Id', async () => {
```

Comment:
Hmm do we plan on re-enabling this test in the future? If not, I suppose we should remove the test since there's no point in keeping a permanently disabled test.

Reply:
Removing the test now.